Ginger/Ginger/AutomatePageLib/AddActionMenu/SharedRepositoryLib/RepositoryItemUsagePage.xaml.cs

```diff
@@ -189,23 +187,100 @@ private void FindUsages()
+                                {
+                                    if (a.ParentGuid == mRepoItem.Guid || a.Guid == mRepoItem.Guid ||
+                                                (mRepoItem.ExternalID != null && mRepoItem.ExternalID != string.Empty && mRepoItem.ExternalID != "0" && a.ExternalID == mRepoItem.ExternalID))
+                                    {
+                                        //skip original if not needed
+                                        if (!mIncludeOriginal)
+                                            if (a.Guid == mOriginalItem.Guid) continue;
+
+                                        Ginger.Repository.RepositoryItemUsage.eUsageTypes type;
+                                        if (a.Guid == mRepoItem.Guid)
+                                            type = RepositoryItemUsage.eUsageTypes.Original;
+                                        else
+                                            type = RepositoryItemUsage.eUsageTypes.Instance;
+
+                                        RepositoryItemUsage itemUsage = new RepositoryItemUsage() { HostBusinessFlow = BF, HostBizFlowPath = Path.Combine(BF.ContainingFolder, businessFlowName), HostActivity = activity, HostActivityName = activity.ActivityName, UsageItem = a, UsageItemName = a.Name, UsageExtraDetails = "Current Value: " + a.Value, UsageItemType = type, Selected = true, Status = RepositoryItemUsage.eStatus.NotUpdated };
+                                        itemUsage.SetItemPartesFromEnum(typeof(VariableBase.eItemParts));
+                                  
+                                        AddUsage(RepoItemUsages, itemUsage);
+                                    }
                                 }
                             }
                         }
-                    }
-                });
+                    });
+                 });
             }
             catch (Exception ex)
             {
                 Reporter.ToUser(eUserMsgKey.GetRepositoryItemUsagesFailed, mRepoItem.GetNameForFileName(), ex.Message);
             }
-        }        
+            finally
+            {
+                StopProcessingIcon();
+                SetStatus("");
+            }
+        }
+
+        private void SetStatus(string txt)
+        {
+            xFindUsageStatusLabel.Dispatcher.BeginInvoke(
+               System.Windows.Threading.DispatcherPriority.Normal,
+                   new Action(
+                       delegate ()
+                       {
+                           xFindUsageStatusLabel.Visibility = Visibility.Visible;
+                           xFindUsageStatusLabel.Content = txt;
+                       }
+           ));
+        }
+
+        private void StartProcessingIcon()
+        {
+            xProcessingIcon.Dispatcher.BeginInvoke(
+               System.Windows.Threading.DispatcherPriority.Normal,
+                   new Action(
+                       delegate ()
```

Comment:
Codacy found an issue: [Remove the parameter list; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6321030)